core-galleon-pack/src/main/resources/feature_groups/elytron-standalone.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<feature-group-spec name="elytron-standalone" xmlns="urn:jboss:galleon:feature-group:1.0">
+
+    <feature spec="subsystem.elytron">
+        <param name="final-providers" value="combined-providers"/>
+        <param name="disallowed-providers" value="[OracleUcrypto]"/>
+        <feature-group name="elytron-app">
+            <include feature-id="subsystem.elytron.security-domain:security-domain=ApplicationDomain">
+                <param name="realms" value="[{realm=ApplicationRealm,role-decoder=groups-to-roles},{realm=local}]"/>
+                <!--
```

Comment:
why is it commented? This looks related to domain mode and could be removed, right?

Reply:
It was commented in the original feature-group: standalone-profile from which I copied the content. That is an alternative syntax to set the attribute. We should keep it.